### ISSUE TITLE
[bug]: Megadropdown trigger focus color

### ISSUE THREAD

**Issue by gfellerph** [swisspost/design-system#4270]
**Describe the bug**
Megadropdown trigger has a blue focus outline while it should be black like everywhere else.

**To Reproduce**
Focus a megadropdown trigger on desktop.

<img width="397" alt="Image" src="https://github.com/user-attachments/assets/b86ef3ef-9308-424e-9e24-19cc6e771f54" />

**Expected behavior**
The focus outline should be black.

<img width="503" alt="Image" src="https://github.com/user-attachments/assets/6abc1a72-4620-41e3-9b48-a29a0f4b1577" />

**Screenshots**

**Desktop (please complete the following information):**

- OS: Mac
- Browser chrome
- Version current
